Updated index.md for Troubleshooting (#76)
vinegarhq/vinegarhq.github.io: src/Troubleshooting/index.md

This specific problem can be the same as many other root causes, a common one is your graphics card not supporting Vulkan, or is missing the graphics libraries required for Wine; you can see [Lutris Docs: Installing Drivers](https://github.com/lutris/docs/blob/master/InstallingDrivers.md)

This may also be your GPU not supporting Vulkan or not supporting modern Vulkan; Use the OpenGL renderer, or set the installed DXVK version to one which includes a legacy version of Vulkan that your GPU supports.

#### "Make sure there are no stable wineservers running"

> You need to end the `wineserver` process. Using System Monitor from many Desktop Environments, or TUI System Monitors like Btop.

Reviewer: This should get into specifics of ESYNC, as this is the cause. Mention `vinegar kill` as well.

Or you can wait until the wineserver process will end.

#### Not launching Roblox Studio

You need to set the DPI to 97. By executing the command `vinegar exec winecfg`, then open the Graphics section and you can change the DPI to 97.

> The only Downside of this is that the Font for Roblox Player will be a bit blurry. Even Numbers for DPI will fix this issue, but Roblox Studio and Player will have a bigger GUI size.

Reviewer: ```suggestion
The only downside of this is that the fonts for Roblox will be a bit blurry, an even number for the DPI would fix this issue, but Roblox will have a bigger interface.
```

#### WebView2 installer stuck on installing

Theres an unknown reason for how this error occured, so unfortunately we can't explain whats the issue. Luckily we do have the solution.

You must change the Windows Version to 7 in `winecfg`.

> You can set the Windows Version to 7 by executing the command `vinegar exec winecfg`. And a GUI will pop up, then you can finally set the Windows Version to 7.

Reviewer: Provide a prebuilt command for winecfg that sets the version to 7 `vinegar exec winecfg /v win7`

> #### Cannot login to Studio or Player

Reviewer: Mention the white screen on login as well

This is because of the horrible way WebView2 runs on Wine.

Login to the [Roblox Website](https://roblox.com) and join a game from there, The login will be synced perfectly with Studio and Player.

#### Bad Performance

There are a few fixes for this, listed here.

You can install the Vulkan Drivers on the 
[Lutris Docs: Installing Drivers](https://github.com/lutris/docs/blob/master/InstallingDrivers.md)

Or you can enable esync/fsync and enable the Vulkan Renderer in the Configuration File of Vinegar (Another great alternative is DXVK which allows more functionality for Studio). You can change 

> [Vinegar Configuration File](https://vinegarhq.github.io/Configuration/index.html) here.

Reviewer: Remove, this deserves its own section in the documentation
